packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -41,7 +41,7 @@ final RegExp _numberedAndroidPlatformRe = RegExp(r'^android-([0-9]+)$');
 final RegExp _sdkVersionRe = RegExp(r'^ro.build.version.sdk=([0-9]+)$');
 
 /// The minimum Android SDK version we support.
-const int minimumAndroidSdkVersion = 25;
+const int minimumAndroidSdkVersion = 28;
```

Comment:
This line doesn't actually seem to cause problems - I'll leave it for a separate PR in case it actually would break anything.  Looks like it just controls what gets written to the AndroidManifest.xml